code/modules/shuttle/shuttles/escape_shuttle.dm

```diff
@@ -76,14 +87,24 @@
 		return
 
 	destination = null
-	if(prob(crash_land_chance))
+	if(prob((EvacuationAuthority.evac_status >= EVACUATION_STATUS_IN_PROGRESS ? crash_land_chance : early_crash_land_chance)))
 		create_crash_point()
 
 	set_mode(SHUTTLE_IGNITING)
 	on_ignition()
 	setTimer(ignitionTime)
 	launched = TRUE
 
+	if(!crash_land) // so doors won't break in space
```

Comment:
Instead of risking the doors opening / not being there in space, you could make it so they can pry the door open, but it closes automatically after some time or during the launch, as a potential idea

Reply:
Yeah I thought of it but then I'd need to make doors unacidable again. Also did you know that unacidable var makes object also unslashable?

Another thing to consider is that slashing is kinda more consistent than do_after pry open, because you can launch the pod as soon as you hear slashing, but with do_after it's a bit more tricky. Doable tho.